configuration/configuration.go

```diff
@@ -16,6 +16,18 @@ func init() {
 	view.Unregister(&view.View{Name: "log_write_delays"})
 }
 
+// Exporter holds exporter configuration.
+// Exporter will be used for exporting data for observer-framework
+// Exporter is grpc-base service
+type Exporter struct {
+	// Address specifies address where exporter server starts
+	Address string `insconfig:":0| exporter-api gRPC server starts on this address"`
```

Comment:
Why not "Listen" like in other binaries?

Reply:
because Listen when server ready to accept something, expose or address when server ready to provide something. that's why.